lib/models.go

```diff
@@ -18,8 +18,11 @@ import (
 // GroupSeparator for group IDs.
 const GroupSeparator = "::"
 
+// RootGroupPath is the id of the root group
```

Comment:
```suggestion
// RootGroupPath is the id of the root group
//
// Note(@mstoykov): the constant shouldn't be used in all tests in order to not couple the tests too much with it.
```

cc @joanlopez

Reply:
What I thought initially was more something like telling the future reader that changing this value would represent a breaking change, from which what you suggested now can be deduced from, I guess.